### ISSUE TITLE
Is one negative control useless for this application?

### ISSUE THREAD

**Issue by lowrda** [benjjneb/decontam#134]
Hi!

I have sequenced milk samples (from rats, small amount) but did not take into account the low-biomass issue when I had them sequenced. This means the company that sequenced them only had 1 negative control. I have 30 samples. Is it even worth trying to use decontam with one negative control sequenced? In the future I will be using 5 or 6, as mentioned in the article. 

Thanks so much!

Dana

**Comment by benjjneb** [comment]
> Is it even worth trying to use decontam with one negative control sequenced?

No. Well, decontam-prevalence is not worth using (the approach that relies on negative controls).

If you have DNA concentration measurements you could try decontam-frequency.

**Comment by lowrda** [comment]
Thanks so much for such a quick response, I'll try and use the DNA concentrations then!
